clients/src/main/java/org/apache/kafka/clients/FetchSessionHandler.java

```diff
@@ -437,5 +443,13 @@ public boolean handleResponse(FetchResponse<?> response) {
     public void handleError(Throwable t) {
         log.info("Error sending fetch request {} to node {}: {}.", nextMetadata, node, t.toString());
```

Comment:
Is warning level more appropriate than info?

Reply:
@noslowerdna Thanks for the review. I am not sure we want to change the log level from `info` to `warn` in all the cases in `Fetcher`. @cmccabe is more familiar with this code, so will be interested to see what he thinks.

Reply:
ok